x-pack/plugins/canvas/public/components/workpad_header/share_menu/flyout/hooks/index.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './use_download_runtime';
```

Comment:
nit: we should get into the habit of naming the exports, unless they're extensive.  It helps to be explicit, so we can track the "contract" the index file provides.